marlowe-playground-client/src/MainFrame/State.purs

```diff
@@ -274,8 +278,33 @@ handleAction s (HaskellAction action) = do
       selectView BlocklyEditor
     _ -> pure unit
 
+handleAction s (JavascriptAction action) = do
+  toJavascriptEditor (JavascriptEditor.handleAction s action)
+  case action of
+    JS.SendResultToSimulator -> do
+      mContract <-
+        peruse
+          ( _javascriptState
+              <<< JS._compilationResult
+              <<< JS._CompiledSuccessfully
+              <<< JSI._result
+          )
+      let
+        contract = case mContract of
+          Just con -> show $ pretty con
+          _ -> ""
+      assign (_simulationState <<< _source) Javascript
+      selectView Simulation
+      void $ toSimulation
```

Comment:
I read this and couldn't see why we were `void`ing out the result. It's because I have expectations that a `toX` function will convert something into a different type, and I couldn't see why we'd convert it only to throw it away.

So I wonder, is there a better naming convention for these, "dispatch to a submodule," functions?

Reply:
Maybe just `s/to/handle`?

Reply:
I don't understand? I want to maintain the option to return something, I guess `handle` might be a bit better but it's marginal

Reply:
It's just the naming that throws me. Entirely subjective and you make the final call. But I read `toX` as a function that turns something into an `X`. Whereas I read `handle` as some kind of, "take care of this action," function.

Reply:
it does turn `m a` into `n a` basically